cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -92,14 +114,17 @@ function FlyoutMenu( element ) {
       this.dispatchEvent( 'expandBegin', { target: this } );
       _isExpanded = true;
       _isAnimating = true;
-      // If transition is not supported, call handler directly (IE9/OperaMini).
-      if ( _transitionEndEvent ) {
-        _contentDom.addEventListener( _transitionEndEvent, _expandEndBinded );
+      if ( _expandTransitionMethod ) {
+        _expandTransitionMethod.call();
```

Comment:
Why are you using `call` here? Is it because `_expandTransitionMethod` is bound to a different scope?